plotly/grid_objs/grid_objs.py

```diff
@@ -148,7 +150,21 @@ def __init__(self, columns_or_json, fid=None):
         ```
         """
         # TODO: verify that columns are actually columns
-        if isinstance(columns_or_json, dict):
+        if pd and isinstance(columns_or_json, pd.DataFrame):
+            column_names = [name for name in columns_or_json.columns]
```

Comment:
🐄 if you *need* this to be a list, why not just `list(columns_or_json.columns)`? If you *don't* need this to be a list, why not just leave it as a pandas array? `column_names = columns_or_json.columns`?